### PR TITLE
Record path to crate root for git deps

### PR DESCRIPTION
**Pull request by illicitonion**
This allows us to generate a strip_prefix directive in the
new_git_repository call.

### CODE REVIEW

**Comment by acmcarther** [comment]
The change looks good to me, but what's the context on the "new_git_repository" call (for me and for future readers)?

**Comment by illicitonion** [comment]
This could be used here: https://github.com/google/cargo-raze/blob/fd59642efab68da8963d7c8540885ba59e7a19cc/impl/src/templates/remote_crates.bzl.template#L16-L26

I'm currently experimenting with a custom renderer, so haven't updated the templates in this repo, but may do so in the future when I wrap up my experiments :)

**Review by acmcarther — APPROVED** [review]
LGTM

File: impl/src/context.rs [hunk 1]
```diff
@@ -66,6 +66,9 @@ impl Default for LicenseData {
 pub struct GitRepo {
   pub remote: String,
   pub commit: String,
+
+  // Directory containing the crate's Cargo.toml file, relative to the git repo root.
```
[the remark this answers is not in the document]
Comment (illicitonion) [reply impl/src/context.rs:70]: Done

**Comment by acmcarther** [comment]
Thanks for putting this together!

**Pull request merged**